system/security/securesocket/securesocket.cpp

```diff
@@ -1370,10 +1478,7 @@ class CSecureSocketContext : implements ISecureSocketContext, public CInterface
         }
     }
 
-    ~CSecureSocketContext()
-    {
-        SSL_CTX_free(m_ctx);
-    }
+    ~CSecureSocketContext() { }
```

Comment:
Should we leave an explicit empty dtor ?

Reply:
no need, better to remove it.

Reply:
ok, removed.